ironfish/src/storage/database/encoding.ts

```diff
@@ -25,9 +25,9 @@ export class StringEncoding<TValues extends string = string>
   }
 }
 
-export class BigIntEncoding implements IDatabaseEncoding<bigint> {
-  serialize = (value: bigint): Buffer => Buffer.from(value.toString(), 'utf8')
-  deserialize = (buffer: Buffer): bigint => BigInt(buffer.toString('utf8'))
+export class NumberEncoding implements IDatabaseEncoding<number> {
+  serialize = (value: number): Buffer => Buffer.from(value.toString(), 'utf8')
```

Comment:
You can also just use a faster write the number straight to the buffer now that its just a number

```js
const b = Buffer.alloc(64)
b.writeDoubleBE(value)
return b
```

Reply:
Updated!